app/gui/view/graph-editor/src/selection/bounding_box.rs

```diff
@@ -74,6 +76,45 @@ impl BoundingBox {
         self.bottom -= size / 2.0;
         self.top += size / 2.0;
     }
+
+    /// Return the x position of the left boundary.
+    pub fn left(&self) -> f32 {
+        self.left
+    }
+
+    /// Return the x position of the right boundary.
+    pub fn right(&self) -> f32 {
+        self.right
+    }
+
+    /// Return the y position of the top boundary.
+    pub fn top(&self) -> f32 {
+        self.top
+    }
+
+    /// Return the y position of the bottom boundary.
+    pub fn bottom(&self) -> f32 {
+        self.bottom
+    }
+
+    /// Expand the boundaries to make them contain all points belonging to the bounding box passed
+    /// as an argument.
+    pub fn grow_to_include(&mut self, other: &BoundingBox) {
+        self.left = self.left.min(other.left);
+        self.right = self.right.max(other.right);
+        self.bottom = self.bottom.min(other.bottom);
+        self.top = self.top.max(other.top);
```

Comment:
Prelude exports `min` and `max` as global functions, which makes them more convenient to use.